### PR TITLE
Globalization

### PR DESCRIPTION
**Pull request by richardpringle**
Connect to strongloop/loopback/issues/2422

@0candy, please review.

### CODE REVIEW

**Comment by 0candy** [comment]
There is a linting error:
```/home/jenkins/workspace/nb/loopback-swagger~master/5570eb3e/lib/specgen/schema-builder.js
  146:1  error  Line 146 exceeds the maximum line length of 90  max-len

��� 1 problem (1 error, 0 warnings)```

**Comment by richardpringle** [comment]
@0candy did you get a chance to review the rest? I fixed the linting error.

**Comment by 0candy** [comment]
@richardpringle The changes LGTM.

**Comment by richardpringle** [comment]
@davidcheung nice catch

**Comment by richardpringle** [comment]
@slnode test please

File: package.json [hunk 1]
```diff
@@ -37,6 +37,7 @@
     "debug": "^2.2.0",
     "ejs": "^1.0.0",
     "lodash": "^3.10.1",
+    "strong-globalize": "^2.6.0",
```
Comment (superkhau) [inline package.json]: ^2.6.2 but its a nitpick

**Comment by superkhau** [comment]
One minor comment that is just a nit, otherwise LGTM.

File: package.json (hunk 1 above)
Comment (superkhau) [inline package.json]: It's now ^2.6.6

**Comment by richardpringle** [comment]
@0candy, this is just waiting for green on loopback-workspace to merge.

@jannyHou, was that you fixing the timeout issues on workspace?

**Comment by 0candy** [comment]
@richardpringle You can merge this PR as all the test are green, only the dependents are failing. I am working on the timeout issue on workspace. Thanks!

**Comment by jannyHou** [comment]
@richardpringle I fixed some in loopback, but not in workspace. Miroslav mentioned he will take  a look why test run so slowly recently.
ATM, "fix" just means increasing timeout, but that's not a long term solution.

**Pull request merged**